_sass/bulma/css/bulma.css

```diff
@@ -388,8 +388,8 @@ a:hover {
 }
```

Comment:
Could you make the changes not in the `_sass/bulma/css/bulma.css` but in `css/custom.scss`?

Reply:
Hi @bebatut I have tried making the changes using the css/custom.scss file but I am unable to find the class needed in the custom.scss file, I'm only able to find it in the _site css folder and the _sass folder. Is their another way I can do this?

Reply:
Everything you put the custom.css file will overwrite the css from Bulma folder. You can then define the class and add the components to overwrite there